features/include/pcl/features/fpfh_omp.h

```diff
@@ -40,91 +40,8 @@
-#include <pcl/features/fpfh.h>
-
-namespace pcl
-{
-  /** \brief FPFHEstimationOMP estimates the Fast Point Feature Histogram (FPFH) descriptor for a given point cloud
-    * dataset containing points and normals, in parallel, using the OpenMP standard.
-    *
-    * \note If you use this code in any academic work, please cite:
-    *
-    *   - R.B. Rusu, N. Blodow, M. Beetz.
-    *     Fast Point Feature Histograms (FPFH) for 3D Registration.
-    *     In Proceedings of the IEEE International Conference on Robotics and Automation (ICRA),
-    *     Kobe, Japan, May 12-17 2009.
-    *   - R.B. Rusu, A. Holzbach, N. Blodow, M. Beetz.
-    *     Fast Geometric Point Labeling using Conditional Random Fields.
-    *     In Proceedings of the 22nd IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS),
-    *     St. Louis, MO, USA, October 11-15 2009.
-    *
-    * \attention
-    * The convention for FPFH features is:
-    *   - if a query point's nearest neighbors cannot be estimated, the FPFH feature will be set to NaN
-    *     (not a number)
-    *   - it is impossible to estimate a FPFH descriptor for a point that
-    *     doesn't have finite 3D coordinates. Therefore, any point that contains
-    *     NaN data on x, y, or z, will have its FPFH feature property set to NaN.
-    *
-    * \author Radu B. Rusu
-    * \ingroup features
-    */
-  template <typename PointInT, typename PointNT, typename PointOutT>
-  class FPFHEstimationOMP : public FPFHEstimation<PointInT, PointNT, PointOutT>
-  {
-    public:
-      using Ptr = shared_ptr<FPFHEstimationOMP<PointInT, PointNT, PointOutT> >;
-      using ConstPtr = shared_ptr<const FPFHEstimationOMP<PointInT, PointNT, PointOutT> >;
-      using Feature<PointInT, PointOutT>::feature_name_;
-      using Feature<PointInT, PointOutT>::getClassName;
-      using Feature<PointInT, PointOutT>::indices_;
-      using Feature<PointInT, PointOutT>::k_;
-      using Feature<PointInT, PointOutT>::search_parameter_;
-      using Feature<PointInT, PointOutT>::input_;
-      using Feature<PointInT, PointOutT>::surface_;
-      using FeatureFromNormals<PointInT, PointNT, PointOutT>::normals_;
-      using FPFHEstimation<PointInT, PointNT, PointOutT>::hist_f1_;
-      using FPFHEstimation<PointInT, PointNT, PointOutT>::hist_f2_;
-      using FPFHEstimation<PointInT, PointNT, PointOutT>::hist_f3_;
-      using FPFHEstimation<PointInT, PointNT, PointOutT>::weightPointSPFHSignature;
-
-      using PointCloudOut = typename Feature<PointInT, PointOutT>::PointCloudOut;
-
-      /** \brief Initialize the scheduler and set the number of threads to use.
-        * \param[in] nr_threads the number of hardware threads to use (0 sets the value back to automatic)
-        */
-      FPFHEstimationOMP (unsigned int nr_threads = 0) : nr_bins_f1_ (11), nr_bins_f2_ (11), nr_bins_f3_ (11)
-      {
-        feature_name_ = "FPFHEstimationOMP";
+PCL_DEPRECATED_HEADER(1, 13, "Use <pcl/features/fpfh.h> instead.")
```

Comment:
The number here is for removal. Release in 1.12 and removal in 1.13 doesn't sound great. Make it the 1.14 at least